cli/docker_validator.py

```diff
@@ -10,7 +11,8 @@
 
 logger = logging_config.get_logger(__name__)
 
-container_image = 'eva_sub_cli'
+container_image = 'ebivariation/eva-sub-cli'
+container_tag = 'v0.0.1.dev0'
```

Comment:
I am wondering if it is advisable to hardcode this in source code as opposed to supplying this in a config file. This way  we  don't have to recompile when the container version changes...

Reply:
At this point I think it is fine to have it hardcode in the code since we  probably want to ensure tight coupling between the python and the docker image version. 
It could be hard coded elsewhere than in this file but I think it should be under version control.